Applications/src/auto-contrast.cc

```diff
@@ -0,0 +1,104 @@
+/*=========================================================================
+
+  Library   : Image Registration Toolkit (IRTK)
+  Module    : $Id$
+  Copyright : Imperial College, Department of Computing
+              Visual Information Processing (VIP), 2008 onwards
+  Date      : $Date$
+  Version   : $Revision$
+  Changes   : $Author$
+Copyright (c) 1999-2014 and onwards, Imperial College London
+All rights reserved.
+See LICENSE for details
+
+=========================================================================*/
+
+#include "mirtk/Common.h"
+#include "mirtk/Options.h"
+
+#include "mirtk/ImageReader.h"
+#include "mirtk/IOConfig.h"
+#include "mirtk/GenericImage.h"
+
+using namespace mirtk;
+
+char *output_name = NULL;
+
+
+void PrintHelp(const char *name)
+{
+  cout << endl;
+  cout << "Usage: " << name << " <image> <output> [options]" << endl;
+  cout << endl;
+  cout << "Description:" << endl;
+  cout << "  This program auto contrast an image." << endl;
```

Comment:
Maybe detail here what computation the program actual performs, i.e., `output = mean + 3 * sdev`.